### PR TITLE
PetabImporter.create_startpoint_method - kwargs, not-None

### PR DESCRIPTION
**Pull request by dweindl**
* Add kwarg forwarding to PetabImporter.create_startpoint_method (to use checked startpoints more conveniently)
* Always return a startpoint method

### CODE REVIEW

File: pypesto/petab/importer.py [hunk 1]
```diff
@@ -568,24 +568,27 @@ def create_prior(self) -> Union[NegLogParameterPriors, None]:
...
         """Create a startpoint method.
 
-        If the PEtab problem specifies an initializationPrior. Returns None,
-        if no initializationPrior is specified.
+        If the PEtab problem specifies an ``initializationPrior``. Returns
+        ``None``, if no ``initializationPrior`` is specified.
+
+        Parameters
+        ----------
+        *+kwargs: Additional keyword arguments passed on to
```
Comment (dweindl) [inline pypesto/petab/importer.py:581]:
```suggestion
        **kwargs: Additional keyword arguments passed on to
```

File: pypesto/petab/importer.py [hunk 2]
```diff
@@ -568,24 +568,24 @@ def create_prior(self) -> Union[NegLogParameterPriors, None]:
         else:
             return None
 
-    def create_startpoint_method(self) -> Union[StartpointMethod, None]:
+    def create_startpoint_method(
+        self, **kwargs
+    ) -> Union[StartpointMethod, None]:
```
Comment (yannikschaelte) [inline pypesto/petab/importer.py:573]:
```suggestion
    ) -> StartpointMethod:
```

**Review by yannikschaelte — APPROVED** [review]
PEtab then internally handles mapping from initialization or parameter priors. Looks reasonable to me.

**Comment by dweindl** [comment]
An independent issue is, that changing the bounds on an PetabImporter-generated Problem, will have no effect on the startpoint sampling, as the bounds of the original PEtab problem will be used. Not sure if bug or feature.

**Comment by yannikschaelte** [comment]
Certainly a feabug. Initialization prior precedes bounds when it comes to start point sampling. If this is clearly stated, then it should be fine?

**Comment by FFroehlich** [comment]
> An independent issue is, that changing the bounds on an PetabImporter-generated Problem, will have no effect on the startpoint sampling, as the bounds of the original PEtab problem will be used. Not sure if bug or feature.

sounds like we need a PetabProblem class with some frozen member attributes? Sounds like a nice opportunity to shoot yourself in the foot otherwise.

**Review by FFroehlich — APPROVED** [review]

**Comment by FFroehlich** [comment]
Also, why am I a codeowner here?

**Pull request merged**